eventmesh-operator/config/samples/eventmesh_operator.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eventmesh-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: eventmesh-operator
+  template:
+    metadata:
+      labels:
+        name: eventmesh-operator
+    spec:
+      serviceAccountName: eventmesh-operator
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: eventmesh-operator
+          # image
+          image: alonexc/eventmesh-operator:latest
+          command:
+            - /manager
+          args:
+            - --leader-elect
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            allowPrivilegeEscalation: false
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8081
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+            requests:
+              cpu: 10m
+              memory: 64Mi
```

Comment:
Is that enough?

Reply:
@Pil0tXia On my computer can run normally, in use can be modified according to their own needs.

Reply:
We assume a standard deployment scenario with 3 EventMesh Runtimes and 10 pairs of connectors in the cluster. Users launch 5 pairs of connectors within five minutes from the console. We would like the default configuration to meet the requirements of this scenario.

Reply:
<img width="1575" alt="image" src="https://github.com/apache/eventmesh/assets/91315508/b507cca8-91ee-4f27-89cf-faa060657fd0">
The test has been modified, as shown in the figure, and the deployment can be basically completed in one minute.